composer.json

```diff
@@ -56,6 +56,9 @@
             "build/themes/contrib/{$name}": ["type:drupal-theme"]
         },
         "patches": {
+            "drupal/core": {
+              "https://www.drupal.org/project/drupal/issues/1091852": "https://www.drupal.org/files/issues/2019-11-19/1091852-104.patch"
```

Comment:
Seems this patch is on a super old issue and lots of potential problems that it can cause. I suggest we remove it and rely only on ajax on the form for the dynamic show/hide of fields instead of states.

Reply:
It's removed.